spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosDataPropertiesParser.java

```diff
@@ -36,13 +44,19 @@ protected Properties doParse(String data) throws IOException {
 		StringReader reader = new StringReader(data);
 		BufferedReader br = new BufferedReader(reader);
 
+		String key;
+		String value;
+		String[] sourceArr;
+		String[] valueArr;
 		for (String line = br.readLine(); line != null; line = br.readLine()) {
-			String[] arr = line.split("=");
-			if (arr.length != 2) {
-				throw new IOException("error properties format");
+			sourceArr = line.split("=");
+			if (sourceArr.length < 2) {
+				log.debug("ignore no properties format line :{}", line);
 			}
-
-			properties.put(arr[0], arr[1]);
+			key = sourceArr[0].trim();
+			valueArr = Arrays.copyOfRange(sourceArr, 1, sourceArr.length);
```

Comment:
this a piece of code use `substring()` and `indexOf()` is better done with it

Reply:
> this a piece of code use `substring()` and `indexOf()` is better done with it

how  can I use substring and indexOf at array?

Reply:
> > this a piece of code use `substring()` and `indexOf()` is better done with it
> 
> how can I use substring and indexOf at array?

e.g line.substring(line.indexOf("=") + 1)

Reply:
complete